src/instructions/interrupts.rs

```diff
@@ -148,15 +148,8 @@ pub fn int3() {
 }
 
 /// Generate a software interrupt by invoking the `int` instruction.
```

Comment:
> I'm not sure if this function has to be `unsafe`, but the code previously emitted by the macro also required `unsafe`, so I marked it as `unsafe` for now.

It should definitely still be unsafe, as triggering an arbitrary interrupt can certainly cause UB. We should include a safety section, saying something like:

```
/// ## Safety
///
/// Invoking an arbitrary interrupt is unsafe. It can cause your system to
/// crash if you invoke a double-fault (#8) or machine-check (#18) exception.
/// It can also cause memory/register corruption depending on the interrupt
/// implementation (if it expects values/pointers to be passed in registers).
```